package.json

```diff
@@ -34,6 +34,9 @@
     "prepack": "./scripts/build"
   },
   "dependencies": {
+    "@types/pg": "^7.4.10",
+    "@types/graphql": "^0.8.2",
+    "@types/jsonwebtoken": "<7.2.1",
```

Comment:
because our interfaces expose types that use these types, the Typescript documentation states to bundle them as dependencies.  i was mixed feelings on this!